usaspending_api/agency/views/test.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class EndpointTest(APIView):
+    """blah"""
```

Comment:
I know we've been champions of having as much documentation as possible, but I really question the value of this comment.

Reply:
It's part of the CI checks

$ python manage.py check_for_endpoint_documentation
usaspending_api.agency.views.test.EndpointTest (/api/v2/agency/test/) has no docstring

You do realize this is a temporary endpoint, right?

Reply:
As long as you're aware that it says that I'm happy.